client/src/containers/RegisterContainer/index.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Register from './Register';
+import Register from '../../components/Register';
```

Comment:
why did u change it?

Reply:
Because when i fixed structure client(scaffolding) i forgot to change name of component and  move RegisterContainer to folder 'containers', in this pr i fixed that.